src/utils/entraApp.ts

```diff
@@ -410,5 +412,28 @@ export const entraApp = {
     }
 
     return resolvedApis;
+  },
+  async getAppObjectIdFromAppId(appId: string): Promise<string> {
```

Comment:
Let's change the interface of this helper function a bit to make it more broadly applyable:


```suggestion
  async getAppRegistrationByAppId(appId: string, properties?: string[]): Promise<string> {
```

Let's return the entire object and let's make it possible to $select specific properties.